guides/common/modules/proc_configuring-server-for-use-with-tftp.adoc

```diff
@@ -1,41 +1,53 @@
-[id="configuring-external-tftp_{context}"]
-= Configuring {ProductName} with external TFTP
+[id="configuring-server-for-use-with-tftp"]
+= Configuring {ProductName} for use with tftp
 
 You can configure {ProductName} with external TFTP services.
 
+
+.Prerequisites
+* You configured the TFTP server.
+* You shared the `/var/lib/tftpboot` on the TFTP server with NFS.
+
+
 .Procedure
+. Associate the TFTP service with the appropriate subnets.
+
+. For each affected {SmartProxy}, update the configuration of that {SmartProxy} in the {ProjectWebUI}. See xref:associating-the-tftp-service-with-a-subnet[].
+
 . Create the TFTP directory for NFS:
 +
 [options="nowrap"]
 ----
 # mkdir -p /mnt/nfs/var/lib/tftpboot
 ----
+
 . In the `/etc/fstab` file, add the following line:
 +
 [options="nowrap" subs="+quotes"]
 ----
 _TFTP_Server_IP_Address_:/exports/var/lib/tftpboot /mnt/nfs/var/lib/tftpboot nfs rw,vers=3,auto,nosharecache,context="system_u:object_r:tftpdir_rw_t:s0" 0 0
 ----
+
 . Mount the file systems in `/etc/fstab`:
 +
 [options="nowrap"]
 ----
 # mount -a
 ----
-. Enter the `{foreman-installer}` command to make the following persistent changes to the `/etc/foreman-proxy/settings.d/tftp.yml` file:
+
+. Configure {ProjectServer} or {SmartProxyServer} to use the TFTP server:
 +
 [options="nowrap" subs="+quotes,attributes"]
 ----
 # {foreman-installer} \
```

Comment:
I suppose this is the first time this gets reviewed because it's missing a crucial line: `--foreman-proxy-managed false`. It will work without it, but as a bonus you get a local TFTP server running. Entirely against the point of the whole procedure.

Reply:
I added the option.